it/subscription_services_privacy_notice.md

```diff
@@ -1,65 +1,78 @@
+Inoltre, se hai fornito il consenso alla raccolta dei dati all'interno dell'applicazione, il nostro partner Sentry.io riceverà dati tecnici per il monitoraggio degli errori e delle prestazioni. Per ulteriori informazioni, consulta l'[Informativa sulla privacy di Sentry.io](https://sentry.io/privacy/).
 
-__Dati di rete.__ All'attivazione, Mozilla VPN provvede alla crittografia del tuo traffico internet e lo invia a Mullvad. Quando usi il servizio Mozilla VPN, né Mozilla né il nostro partner Mullvad conservano alcun registro con le tue attività sulla rete. Per ulteriori informazioni, consulta l'[Informativa sulla privacy di Mullvad](https://mullvad.net/help/no-logging-data-policy/).
+__Dati sulla posizione.__ Quando ti iscrivi e utilizzi i servizi, Mozilla riceve il tuo indirizzo IP, che viene memorizzato nei registri del nostro server per 90 giorni; lo utilizziamo per determinare approssimativamente la tua posizione, poiché la disponibilità, i prezzi e le offerte dei Servizi in abbonamento di Mozilla possono variare in base al Paese in cui ti trovi. Lo utilizziamo anche per la prevenzione di frodi e abusi nei nostri prodotti.
+
+__Dati di rete.__ All'attivazione, Mozilla VPN provvede alla crittografia del tuo traffico Internet e lo invia a Mullvad. Quando usi il servizio Mozilla VPN, né Mozilla né il nostro partner Mullvad conservano mai alcun registro con le tue attività sulla rete. Per ulteriori informazioni, consulta l'[Informativa sulla privacy di Mullvad](https://mullvad.net/help/no-logging-data-policy/).
+
+### Mozilla Monitor {: #monitor }
+
+__Dati relativi alle interazioni.__ Mozilla riceve dati sulle tue interazioni con i servizi Per esempio, quando accedi o esci e le preferenze che hai impostato; per saperne di più, consulta i [dati relativi alle interazioni di Monitor](https://dictionary.telemetry.mozilla.org/apps/monitor_frontend). Più generalmente, Mozilla può inoltre usare cookie, informazioni sul dispositivo e indirizzi IP, insieme a GIF trasparenti, cookie e servizi di terzi per per comprendere nell'insieme il modo in cui ti relazioni ai nostri prodotti, servizi, comunicazioni, siti web, campagne online, frammenti di codice, dispositivi e altre piattaforme. Utilizziamo:
+
+* [Have I Been Pwned.](https://haveibeenpwned.com/) Il servizio Have I Been Pwned riceve una versione con hash del tuo indirizzo email per eseguire una scansione laddove i tuoi dati personali potrebbero essere stati compromessi. Have I Been Pwned conserva questi dati personali per continuare a monitorare nuove violazioni; puoi richiedere la rimozione del tuo indirizzo email da Have I Been Pwned tramite il relativo [strumento di cancellazione.](https://haveibeenpwned.com/OptOut).
```

Comment:
```suggestion
* [Have I Been Pwned.](https://haveibeenpwned.com/) Il servizio Have I Been Pwned riceve un “hash” del tuo indirizzo email per eseguire una scansione laddove i tuoi dati personali potrebbero essere stati compromessi. Have I Been Pwned conserva questi dati personali per continuare a monitorare nuove violazioni; puoi richiedere la rimozione del tuo indirizzo email da Have I Been Pwned tramite il relativo [strumento di cancellazione.](https://haveibeenpwned.com/OptOut).
```

"versione con hash" (version with hash) is incorrect. A "hashed version" is hash of the email address, it's not added to the email address.